### ISSUE TITLE
Gfycat in comments not showing correctly.

### ISSUE THREAD

**Issue by MichealKenny** [thomasandersen/gfycat-companion-for-firefox#2]
When Gfycat plays in the comments, this is what I see:

![Playing](http://puu.sh/73P3h.png)

When I pause it, it looks normal:

![Paused](http://puu.sh/73P3V.jpg)

**Comment by thomasandersen** [comment]
Michael
Does this happens every time? What happens when you access and play a video directly?
http://fat.gfycat.com/FineDearestBarb.webm

Could you please try version 0.4a2 and see if the issue still exist!?
http://mr-andersen.no/companion/gfycat-companion-for-firefox-0.4a2.xpi

**Comment by MichealKenny** [comment]
Yeah I can load it directly just fine, updating to 0.4a2 didn't help, honestly it is probably an issue with RES (I'm on Firefox 27.0.1/RES 4.3.1.2 Win 7 64-bit btw) but I just thought I would point it out.

**Comment by thomasandersen** [comment]
Thank you for reporting. Right now I can not reproduce this on this setup (Fx 27, OS X). 
Do you see this issue other places? Like: http://www.w3.org/2010/05/video/mediaevents.html or http://gfycat.com/BarrenJubilantAntarcticgiantpetrel

**Comment by MichealKenny** [comment]
No both of those are fine, I only get the problem when I open I maximise a gfycat link using RES in the comments.
